### PR TITLE
Add Json rules file

### PR DESCRIPTION
**Pull request by ihtsham**
I added a new  JSON rule file

### CODE REVIEW

File: Assignments/JsonRule.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+               ]
+            }
+         }
+      ],
+      "B2":{
+         "Rules":[
+            {
+               "ID":4,
+               "Type":"Range_validation",
+               "Message":"Duration should be between 0 and 60",
```
Comment (nasirminhas) [inline Assignments/JsonRule.json:76]: Please change the range from 60 to 80

File: Assignments/JsonRule.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+               "Type":"Position_On_Land",
+               "Message":"Fishing position is on land",
+               "Params":{
+                  "Latitude":"B2.Latitude",
+                  "Longitude":"B2.Longitude"
+               }
+            },
+            {
+               "ID":9,
+               "Type":"Dependency_validation",
```
Comment (nasirminhas) [inline Assignments/JsonRule.json:96]: Rule ID 9: Please include Norway in the list as well

**Review by nasirminhas — CHANGES_REQUESTED** [review]
Please address the requested changes.

File: Assignments/JsonRule.json (hunk 1 above)
Comment (ihtsham) [reply Assignments/JsonRule.json]: It has been fixed

File: Assignments/JsonRule.json (hunk 2 above)
Comment (ihtsham) [reply Assignments/JsonRule.json:96]: It has been fixed

Comment (nasirminhas) [reply Assignments/JsonRule.json:96]: I am okay with the changes

File: Assignments/JsonRule.json (hunk 1 above)
Comment (nasirminhas) [reply Assignments/JsonRule.json]: Thanks, it is okay now

**Pull request merged**